packages/dds/tree/src/tableSchema.ts

```diff
@@ -1010,7 +1010,7 @@ export namespace TableSchema {
 
 	/**
 	 * A row in a table.
-	 * @remarks Implemented by the schema class returned from {@link TableSchema.(createRow:2)}.
+	 * @remarks Implemented by the schema class returned from {@link TableSchema.(row:2)}.
```

Comment:
For context, this was a bug that I checked in. It was not caught due to the API-Extractor regression this PR is aimed to resolve.